src/main/jbake/content/blog/2017/05/31/hawkular-grafana-ootb.adoc

```diff
@@ -0,0 +1,253 @@
+= Hawkular and Grafana Out of the Box
+Joel Takvorian
+2017-05-31
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, metrics, grafana, docker, vertx
+:figure-caption!:
+
+link:https://github.com/jotak/hawkular-grafana-outofthebox[Hawkular and Grafana Out of the Box] is a new way to quickly setup Hawkular and Grafana together with very little work.
```

Comment:
It renders larger, because it is the first paragraph, which is sort of the abstract. That is default with AsciiDoc